src/xscen/aggregate.py

```diff
@@ -928,16 +928,18 @@ def spatial_mean(  # noqa: C901
 @parse_config
 def produce_horizon(  # noqa: C901
```

Comment:
maybe add a bit more explanation in the docstring that if you dont give `indicators` , `ds` should be an indicators.

Reply:
there could also be a check with a warning. like ds is daily and you didn't give `indicator`. this is could go wrong. `clim_op` wants a indicator as input.

Reply:
forgot about that. in that case, I don't think it's necessary